subprojects/gradle/instrumentation-tests/src/integTest/kotlin/com/avito/instrumentation/executing/TestExecutorIntegrationTest.kt

```diff
@@ -19,16 +19,13 @@ import com.avito.utils.logging.CILogger
 import com.avito.utils.logging.FakeCILogger
 import com.avito.utils.logging.commonLogger
 import org.junit.jupiter.api.Test
-import org.junit.jupiter.api.Timeout
 import org.junit.jupiter.api.io.TempDir
 import java.io.File
-import java.util.concurrent.TimeUnit.SECONDS
 
 internal class TestExecutorIntegrationTest {
 
     private val logger = FakeCILogger()
 
-    @Timeout(10, unit = SECONDS)
```

Comment:
Why did you remove those?

Reply:
because i believe global timeouts is enough, and we don't need to tune it per test

Reply:
I agree in common but in that Test `Timeout` is an intention of our testing we fix timeouts.

Reply:
i found that sometimes this test timings are also fluctuate and could run over 30sec.
I don't know what is the correct value here, so just removed it in favor of global timeouts

Reply:
In my opinion, this test should take 1 sec maximum. We could pairly fix this strange behavior